### PR TITLE
Quaternion Arcball example update with multiple methods for deriving position

### PR DESCRIPTION
**Pull request by NickHardeman**
quaternionArcballExample

### CODE REVIEW

**Comment by ofTheo** [comment]
thanks @NickHardeman 
do you think a higher ofSetCircleResolution might look better?

( also makes me think that a mode where circle resolution is tied to radius could be a nice option down the line )

**Comment by NickHardeman** [comment]
oh yeah, probably would look much nicer

**Comment by NickHardeman** [comment]
Screen Shot 2022-09-20 at 4 38 14 PM

File: examples/3d/quaternionArcballExample/src/ofApp.cpp [hunk 1]
```diff
@@ -63,9 +163,13 @@ void ofApp::mouseDragged(int x, int y, int button){
...
-	glm::quat xRot = glm::angleAxis(ofDegToRad(y-lastMouse.y)*dampen, glm::vec3(-1,0,0));
-	curRot = xRot * yRot * curRot;
+	
+	xspeed = ofLerp(xspeed, (x-lastMouse.x) * dampen, 0.1);
+	yspeed = ofLerp(yspeed, (y-lastMouse.y) * dampen, 0.1);
+	
+//	glm::quat yRot = glm::angleAxis(ofDegToRad(x-lastMouse.x)*dampen, glm::vec3(0,1,0));
+//	glm::quat xRot = glm::angleAxis(ofDegToRad(y-lastMouse.y)*dampen, glm::vec3(-1,0,0));
+	//curRot = xRot * yRot * curRot;
     lastMouse = mouse;
```
Comment (ofTheo) [inline examples/3d/quaternionArcballExample/src/ofApp.cpp:173]: Oops - one more thing. 
Might be good to remove the commented out lines ( 170 - 171 ) and update the comments above?

Comment (NickHardeman) [reply examples/3d/quaternionArcballExample/src/ofApp.cpp:171]: Ok, cleaned up a bit here https://github.com/openframeworks/openFrameworks/pull/7090/commits/a87b958e560cf93a68231f73ae8b1d57448fa04b

**Comment by ofTheo** [comment]
Amazing - thanks @NickHardeman !

**Pull request merged**